lang/java/avro/src/main/java/org/apache/avro/LogicalTypes.java

```diff
@@ -267,6 +281,12 @@ public static LocalTimestampMicros localTimestampMicros() {
     return LOCAL_TIMESTAMP_MICROS_TYPE;
   }
 
+  private static final LocalTimestampMicros LOCAL_TIMESTAMP_NANOS_TYPE = new LocalTimestampMicros();
+
+  public static LocalTimestampMicros localTimestampNanos() {
+    return LOCAL_TIMESTAMP_NANOS_TYPE;
+  }
+
```

Comment:
Shoudln't it be LocalTimestampNanos class here?

Reply:
Yes, you're correct! Do you want to raise the PR?

Reply:
Done at https://github.com/apache/avro/pull/2706